Readme.md

```diff
@@ -37,48 +37,12 @@ NumberList ::= number (sep number)*
 Token input using BNF
```

Comment:
It isn't clear what this means - This is the input being fed to the lexer, and afterwards it shows the tokens. Remember that someone reading the readme is likely not already familiar with these concepts, so make sure the language you're using is precise and will make sense to someone who's just learning what this library is about.